### PR TITLE
81 meds enhancement

### LINKED ISSUE #58: Cancer Therapy Filter
+ Drug
        Axitinib
        Bevacizumab
        Everolimus
        IL-2
        Interferon
        Ipilimumab
        Nivolumab
        Pazopanib
        Sorafenib
        Sunitinib
        Temsirolimus
    + Line
        1
        2
        3
        4
        5
        6
        7
        8
        9
    + Duration
        0 - 3 months
        3 - 6 months
        6 - 9 months
        9 - 12 months
        12 - 18 months
        18 - 24 months
        24 - 30 months
        30 - 36 months
        36 - 48 months
        48+ months

### PR DESCRIPTION
**Pull request by SenseCode**
Close #58, #81, #82...

### CODE REVIEW

File: src/encoded/types/patient.py [hunk 1]
```diff
@@ -200,6 +202,54 @@ def consent(self, request, consent):
...
+        
+        for object in medication:
+            medication_object = request.embed(object, '@@object')
+            date_format="%Y-%m-%d"
+            start_date=datetime.strptime(medication_object['start_date'],date_format )
+            end_date=datetime.strptime(medication_object['end_date'],date_format )
+            medication_duration=math.ceil((end_date-start_date).days/30)
+            duration_list=[] 
+            if 0<medication_duration<=3:
+                duration_list_item="1-3 month"
```
Comment (mingjiecn) [inline src/encoded/types/patient.py]: In the issue, duration range starts at "0 - 3 month", not "1 - 3 month".
Do we have a definition on duration in month so we can implement code accordingly?
Also is it accurate to use math.ceil() function to modify the duration time before comparing?

File: src/encoded/static/components/medicationChart.js [hunk 2]
```diff
@@ -22,16 +25,38 @@ class MedicationChart extends React.Component {
...
+    this.zoomOut = this.zoomOut.bind(this);
+
   }
   render() {
     return (
       <div className="flex-container" >
+        <div className="chart-menu" >
+          <div className="chartboxes pt-2" >
+            <div className="mb-2">
+              <button type="button" className="btn btn btn-danger" onClick={this.zoomIn} title="Zoom in" aria-label="Zoom in"><FontAwesomeIcon icon={faSearchPlus} size="2x" /></button>
```
Comment (mingjiecn) [inline src/encoded/static/components/medicationChart.js]: Different className for zoom in and zoom out buttons compare to zoom in and zoom out button in patientChart.js. We need to discuss what className to use to keep consistency.

File: src/encoded/types/patient.py (hunk 1 above)
Comment (vsmalladi) [reply src/encoded/types/patient.py]: Update range to be 
0 <= duration < 3
3 <= duration < 6

File: src/encoded/static/components/medicationChart.js (hunk 2 above)
Comment (vsmalladi) [reply src/encoded/static/components/medicationChart.js]: Add to Radiation and standardize across all objects

**Review by vsmalladi — CHANGES_REQUESTED** [review]
Order meds by first date taken and not medication name

**Comment by mingjiecn** [comment]
Label is overlapping when there is only one drug.

File: src/encoded/schemas/patient.json [hunk 3]
```diff
@@ -134,7 +140,10 @@
         "germline.target": 1.0,
         "germline.significance": 1.0,
         "radiation.site_general": 1.0,
-        "radiation.treatment_intention": 1.0
+        "radiation.treatment_intention": 1.0,
+        "medications.name":1.0,
+        "medications.start_date":1.0,
```
Comment (vsmalladi) [inline src/encoded/schemas/patient.json]: Remove Start date and end-date

**Review by vsmalladi — CHANGES_REQUESTED** [review]
Add in sort by duration number.

**Review by vsmalladi — APPROVED** [review]

**Pull request merged**